src/main/java/com/hillel/items_exchange/dto/PhoneDto.java

```diff
@@ -0,0 +1,26 @@
+package com.hillel.items_exchange.dto;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+import javax.validation.constraints.NotNull;
+import javax.validation.constraints.Pattern;
+import javax.validation.constraints.PositiveOrZero;
+
+@NoArgsConstructor
+@AllArgsConstructor
+@Getter
+@Setter
+public class PhoneDto {
+    @PositiveOrZero(message = "{invalid.id}")
+    private long id;
+    @NotNull(message = "{invalid.not-null}")
+    private Boolean show;
+    @Pattern(regexp = "^\\s*(?<country>\\+?\\d{1,3})[-. (]*(?<area>\\d{3})[-. )]*(?<number>\\d{3}[-. ]*\\d{2}[-. ]*\\d{2})\\s*$",
```

Comment:
What you think about extraction regular expressions to property file? It seems to me, this will give quick access to them in case of changes

Reply:
I tried to find out about it, but was unsuccessful. And our project does not have any examples. So I did it according to existing code. Ok, I agree with you, and I will certain try to figure out it again